javaagent-tooling/src/main/java/io/opentelemetry/javaagent/tooling/config/OtlpProtocolPropertiesSupplier.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.tooling.config;
+
+import com.google.auto.service.AutoService;
+import io.opentelemetry.sdk.autoconfigure.spi.AutoConfigurationCustomizer;
+import io.opentelemetry.sdk.autoconfigure.spi.AutoConfigurationCustomizerProvider;
+
+@AutoService(AutoConfigurationCustomizerProvider.class)
+public class OtlpProtocolPropertiesSupplier implements AutoConfigurationCustomizerProvider {
+
+  @Override
+  public void customize(AutoConfigurationCustomizer autoConfigurationCustomizer) {
+    autoConfigurationCustomizer.addPropertiesSupplier(ConfigurationFile::getProperties);
+    autoConfigurationCustomizer.addPropertiesCustomizer(new OtlpProtocolConfigCustomizer());
```

Comment:
try to use `addPropertiesSupplier` here instead

Reply:
but parameter expect an Map<String, String> though